nyc_data/ppe/views.py

```diff
@@ -191,7 +196,7 @@ class UploadContext(NamedTuple):
     import_in_progress: Optional[str] = None
 
 
-class Upload(View):
+class Upload(LoginRequiredMixin, View):
```

Comment:
logins also required on the verify & cancel import view (especially since imports use sequential ids)